docs/snippets/prerequisites.md

```diff
@@ -11,7 +11,7 @@ Before installing Knative, you must meet the following prerequisites:
 
     - If you have only one node in your cluster, you need at least 6&nbsp;CPUs, 6&nbsp;GB of memory, and 30&nbsp;GB of disk storage.
     - If you have multiple nodes in your cluster, for each node you need at least 2&nbsp;CPUs, 4&nbsp;GB of memory, and 20&nbsp;GB of disk storage.
-- You have a cluster that uses Kubernetes v1.18 or newer.
+- You have a cluster that uses a supported Kubernetes version. Refer to the [Release Versioning Principles](https://github.com/knative/community/blob/main/mechanics/RELEASE-VERSIONING-PRINCIPLES.md){_blank} page for more information.
```

Comment:
What does "_blank" mean here? First time I see it.

Reply:
I'm testing out GitHub copilot and.. well I might have to turn it off lol. Sometimes it slips in odd things without me noticing. It's been removed [5f8771d](https://github.com/knative/docs/pull/4196/commits/5f8771d0b8a9e0c54d74bb4686714d7d704fa774)